webpack.config.js

```diff
@@ -10,5 +10,17 @@ module.exports = {
     globalObject: "this"
   },
   mode: "production",
-  devtool: "source-map"
+  devtool: "source-map",
+    optimization: {
+    minimizer: [
+      new TerserPlugin({
+        terserOptions: {
+          output: {
```

Comment:
Could you try removing this `output: {}` and replace it with the inner `keep_classnames: true`?

Reply:
Yeah sure

Reply:
I have made the following changes